cmd/konnector/cmd/konnector.go

```diff
@@ -78,11 +83,23 @@ func New(ctx context.Context) *cobra.Command {
 
 			logger.Info("trying to acquire the lock")
 			lock := NewLock(config.KubeClient, options.LeaseLockNamespace, options.LeaseLockName, options.LeaseLockIdentity)
-			runLeaderElection(ctx, lock, options.LeaseLockIdentity, func(ctx context.Context) {
+
+			le := makeLeaderElectorOrDie(ctx, lock, options.LeaseLockIdentity, func(ctx context.Context) {
 				logger.Info("starting konnector controller")
 				err = prepared.Run(ctx)
 			})
 
+			hz := leaderelection.NewLeaderHealthzAdaptor(LeaderElectionTimeout)
+			hz.SetLeaderElection(le)
+
+			status := healthz.NamedCheck("status", healthz.PingHealthz.Check)
```

Comment:
*Nitpick:* This seems superfluous to me, since `PingHealthz` check is a no-op. Wdyt?

Reply:
removing